src/Symfony/Bundle/TwigBundle/DependencyInjection/Configuration.php

```diff
@@ -35,11 +35,8 @@ public function getConfigTreeBuilder()
         $rootNode
             ->children()
                 ->scalarNode('exception_controller')
-                    ->defaultValue(static function () {
-                        @trigger_error('Relying on the default value ("twig.controller.exception::showAction") of the "twig.exception_controller" configuration option is deprecated since Symfony 4.4, set it to "null" explicitly instead, which will be the new default in 5.0.', E_USER_DEPRECATED);
-
-                        return 'twig.controller.exception::showAction';
-                    })
+                    ->defaultValue('twig.controller.exception::showAction')
+                    ->setDeprecated('The "%path%.%node%" configuration key has been deprecated in Symfony 4.4, use "framework.error_controller" configuration key instead.')
```

Comment:
Can you explain the upgrade path? If people do not set this deprecated option, it will use the default. So it needs to take precedence over the new error rendering for BC. So I would need to disable this by setting it to null so the new approach takes over? But then I would still get the deprecation message.

Reply:
I'm still working on this, related to https://github.com/symfony/symfony/pull/33271#discussion_r316328299

Reply:
See https://github.com/symfony/symfony/pull/33271#issuecomment-524547139